lib/luaxtable.c

```diff
@@ -56,8 +56,9 @@ static int luaxtable_docall(lua_State *L, luaxtable_t *xtable, const char *op, i
 
 	lua_insert(L, base + 1); /* op */
 	lua_pop(L, 1); /* table */
+	lua_pushstring(L, info->userdata); /* userdata */
```

Comment:
I would rename it.. `userdata` clashes with Lua's userdada and make it a bit confusing

Reply:
Can I rename it to `userargs` ?

Reply:
yup.. just fine for me..